### ISSUE TITLE
swift build error doesn't show diagnostics

### ISSUE THREAD

**Issue by haifengkao** [vknabel/vscode-swift-development-environment#57]
**Expected**
The diagnostics shows the location of error 

**Actual**
The diagnostics is empty

when swift build failed, `stderr` is empty string
but `stdout` has all the error messages 
if I remove `if (stderr) `, the diagnostics displays correctly.

```
// SwiftTool.js
sb.on("exit", function (code, signal) {
        clientMain_1.trace(`***swift build command exited*** code: ${code}, signal: ${signal}`);
        clientMain_1.dumpInConsole("\n");
        clientMain_1.diagnosticCollection.clear();

        if (stderr) {
            dumpDiagnostics();
        }
```

**Comment by vknabel** [comment]
@haifengkao Good catch! Would you like to create a PR for this?

**Comment by vknabel** [comment]
Has been fixed with latest release [2.8.2](https://github.com/vknabel/vscode-swift-development-environment/releases/tag/2.8.2)

**Comment by haifengkao** [comment]
too bad I didn't see your comment earlier :(
